airbyte-api/src/main/openapi/config.yaml

```diff
@@ -1824,6 +1936,10 @@ components:
         prefix:
           type: string
           description: Prefix that will be prepended to the name of each stream when it is written to the destination.
+        operationIds:
+          type: array
+          items:
+            $ref: "#/components/schemas/OperationId"
```

Comment:
From this, I would imagine the front-end needs to create operations separately using the API `  /v1/operations/create:` etc 

Once all of the single operations are configured, then it can save and submit the connection components with the proper operation ids as a sequence

Is that reasonable?